Pipfile

```diff
@@ -14,4 +14,4 @@ requests = "*"
 xlsx2csv = "*"
 
 [requires]
-python_version = "3.7"
+python_version = "3.6"
```

Comment:
Is there a reason for the Python 3.6 downgrade? As far as I know Python 3.7 doesn't really add any functionality that we need, but I was just wondering.
Also, 3.8 was recently released and it adds some nice stuff like a walrus operator and a `=` sign in f-strings (prints more information on a variable, nice for debugging).

Reply:
Ubuntu 18.04 LTS is still using Python 3.6 :(